### ISSUE TITLE
Support for composite multiple images

### ISSUE THREAD

**Issue by vntrungld** [vseventer/sharp-cli#73]
How to composite multiple images by a single command? I'm making a PHP SDK for Sharp using this project so maybe we need a solution for composite multiple images like Imagick.

**Comment by vseventer** [comment]
Can you show me what you tried? Although I have not tried this, you should be able to compose multiple images using something like `sharp composite <image1> -- composite <image2>`.

**Comment by vntrungld** [comment]
Hi @vseventer, sorry for the late response
The command I've tried is:
```
sharp -i background-min.png -o result-min.png -f png -- composite face-min.png --gravity southeast -- composite flower-min.png --gravity southeast
```
I tried to put **face** and **flower** on the **background** but the result only have the **flower** and **background**. Look like the command only takes the latest composite image
![background-min](https://user-images.githubusercontent.com/19848743/191159361-38075bed-4e7a-4c83-82d2-5482102b6ba4.png)
![face-min](https://user-images.githubusercontent.com/19848743/191159370-47ef672c-ef32-4f95-9da5-78e31194b58a.png)
![flower-min](https://user-images.githubusercontent.com/19848743/191159372-08075af3-8d48-43be-b7ce-cbf5401bec68.png)
![result-min](https://user-images.githubusercontent.com/19848743/191159376-5fc44a1c-0caa-4001-94d9-70f16dd62f8d.png)

**Comment by vseventer** [comment]
Thanks for providing this - while Sharp allows compositing multiple images, the CLI is not passing the images in the right format.

Since this likely affects other operations too, I got to dig a bit deeper and figure out a solution that works for all affected areas.

**Comment by vseventer** [comment]
Fixed in #75 - hoping to complete that PR soon and release a new major version.

In your case, usage will be:
```bash
sharp -i background-min.png -o result-min.png -f png composite face-min.png flower-min.png --gravity southeast
```

**Comment by vseventer** [comment]
Fixed in v4.0.0, now available.
